types/event.model.ts

```diff
@@ -0,0 +1,40 @@
+import { Crew, Team } from './crew.model';
+import { DisplayStatus } from './display-status.model';
+import { Round } from './round.model';
+
+export interface Event {
+	id: string;
+	regatta_id: string;
+	cost_id: string;
+	parent: null; // Wat is de 'parent'?
+	event: number;
+	number: number; // Is dit hetzelfde als 'event'?
+	day: Date; // Wordt een event op alleen een dag gedaan? Is dat anders dan de regatta?
+	code: string; 
+	name: string; // Is dit hetzelfde als 'name'?
```

Comment:
Mogelijk klopt de comment hier niet

Reply:
Ah ja, ik bedoelde 'code'

Reply:
Maar ik neem aan dat het dan hetzelfde geval is, dus dat het niet hetzelfde hoeft te zijn

Reply:
Dat klopt ja